app/ptxboa_functions.py

```diff
@@ -336,3 +339,213 @@ def write_info(info):
+    st.markdown(data["h2_trade_characteristics"])
+    st.markdown(f"*Source: {data['source_h2_trade_characteristics']}*")
+
+    st.markdown("**LNG import terminals:**")
+    st.markdown(data["lng_import_terminals"])
+    st.markdown(f"*Source: {data['source_lng_import_terminals']}*")
+
+    st.markdown("**H2 pipeline projects:**")
+    st.markdown(data["h2_pipeline_projects"])
+    st.markdown(f"*Source: {data['source_h2_pipeline_projects']}*")
+
+
+def create_fact_sheet_supply_country(context_data: dict, country_name: str):
+    """Display information on a chosen supply country."""
+    df = context_data["supply"]
+    data = df.loc[df["country_name"] == country_name].iloc[0].to_dict()
+
+    st.subheader(f"Fact sheet for {country_name}")
+    text = (
+        "**Technical potential for renewable electricity generation:**\n"
+        f"- {data['source_re_tech_pot_EWI']}: "
+        f"\t{data['re_tech_pot_EWI']:.0f} TWh/a\n"
+        f"- {data['source_re_tech_pot_PTXAtlas']}: "
+        f"\t{data['re_tech_pot_PTXAtlas']:.0f} TWh/a\n"
+    )
+
+    st.markdown(text)
+
+    text = (
+        "**LNG infrastructure:**\n"
+        f"- {data['lng_export']} export terminals\n"
+        f"- {data['lng_import']} import terminals.\n\n"
+        f"*Source: {data['source_lng']}*"
+    )
+
+    st.markdown(text)
+
+    st.write("TODO: CCS pot, elec prices, H2 strategy")
+
+
+def create_fact_sheet_certification_schemes(context_data: dict):
+    """Display information on a chosen certification scheme."""
+    df = context_data["certification_schemes"]
+    helptext = "Select the certification scheme you want to know more about."
+    scheme_id = st.selectbox("Select scheme:", df["ID"], help=helptext)
+    data = df.loc[df["ID"] == scheme_id].iloc[0].to_dict()
+
+    st.header(data["name"])
+
+    st.markdown(data["description"])
+
+    st.subheader("Characteristics")
+
+    st.markdown(
+        f"- **Relation to other standards:** {data['relation_to_other_standards']}"
+    )
+    st.markdown(f"- **Geographic scope:** {data['geographic_scope']}")
+    st.markdown(f"- **PTXBOA demand countries:** {data['ptxboa_demand_countries']}")
+    st.markdown(f"- **Labels:** {data['label']}")
+    st.markdown(f"- **Lifecycle scope:** {data['lifecycle_scope']}")
+
+    st.subheader("Scope")
+    st.markdown("- **Emissions:**")
+    st.markdown(data["scope_emissions"])
+
+    st.markdown("- **Electricity:**")
+    st.markdown(data["scope_electricity"])
+
+    st.markdown("- **Water:**")
+    st.markdown(data["scope_water"])
+
+    st.markdown("- **Biodiversity:**")
+    st.markdown(data["scope_biodiversity"])
+
+    st.markdown("- **Other:**")
+    st.markdown(data["scope_other"])
+
+    st.subheader("Sources")
+    st.markdown(data["sources"])
+
+
+def create_content_sustainability(context_data: dict):
+    """Display information on sustainability issues."""
+    df = context_data["sustainability"]
+    st.image(Image.open("static/sustainability.png"))
```

Comment:
you can directly use the local image path instead of opening a pilliow.Image object

```suggestion
    st.image("static/sustainability.png")
```